status-server/package.json

```diff
@@ -1,8 +1,14 @@
 {
+  "scripts": {
+    "start": "nodemon ./src/index.js"
```

Comment:
Lets also change the README.md in this pull request to update the instructions for starting and stopping the server